components/library/objfw/Makefile

```diff
@@ -29,12 +29,12 @@ USE_PARALLEL_BUILD = yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME = objfw
-COMPONENT_VERSION = 1.2.4
+COMPONENT_VERSION = 1.3
 COMPONENT_REVISION = 1
 COMPONENT_SUMMARY = A portable, lightweight framework for the Objective-C language
```

Comment:
Please remove COMPONENT_REVISION if you change COMPONENT_VERSION.

Reply:
Done.